Content.Shared/Execution/ExecutionComponent.cs

```diff
@@ -84,4 +84,13 @@ public sealed partial class ExecutionComponent : Component
     /// </summary>
     [DataField]
     public bool Executing = false;
-}
\ No newline at end of file
+
+    // Goobstation start
+
+    /// <summary>
+    /// if when this is used for execution then the head be removed from the shoulders
+    /// </summary>
+    [DataField]
+    public bool Decapitation;
```

Comment:
This needs to be set to false by default if you're checking it in the system code.
```suggestion
    [DataField]
    public bool Decapitation = false;
```

Reply:
bools are set to false by defult, but ok